SETUP.md

```diff
@@ -397,7 +397,7 @@ You can then open the Jupyter notebook server at http://localhost:8888
 
 The process of making a new release and publishing it to pypi is as follows:
 
-First make sure that the tag that you want to add, e.g. `0.6.0`, is added in [recommenders.py/__init__.py](recommenders.py/__init__.py). Follow the [contribution guideline](CONTRIBUTING.md) to add the change.
+First make sure that the tag that you want to add, e.g. `0.6.0`, is added in [recommenders.py/`__init__.py`](recommenders.py/__init__.py). Follow the [contribution guideline](CONTRIBUTING.md) to add the change.
```

Comment:
```
[recommenders.py/`__init__.py`](recommenders.py/__init__.py)
```

should be

```
[`recommenders/__init__.py`](recommenders/__init__.py)
```

Reply:
fixed in https://github.com/microsoft/recommenders/pull/1606/commits/48aab4792dee0c9871d2d141122b0f45f4b34453